lib/resources/TransferReversals.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+
+/**
+ * TransferReversals is a unique resource in that, upon instantiation,
```

Comment:
Ah, I see. Basically because in order to instantiate them, you need to pass in a transfer ID, when calling methods on them, you don't need to pass the transfer ID again, correct?